### ISSUE TITLE
Page status incorrectly positioned when scrolling navigation sidebar

### ISSUE THREAD

**Issue by Enn83** [squidfunk/mkdocs-material#6910]
### Bug description

When I have many page in the navigation sidebar (or when the window is shrink), the page status hover is incorrectly positionned.

For example, on the famous **Material for MkDocs** website:
![image](https://github.com/squidfunk/mkdocs-material/assets/46783975/6e11152d-0a7b-40c3-be0f-fb521f3409d6)

You can see the `Recently added` tooltip is positionned like if the navigation sidebar was not scrolled.

### Related links

- [An example of page with a Page status in the nav sidebar (Material for MkDocs)](https://squidfunk.github.io/mkdocs-material/getting-started/)

### Reproduction

[9.5.13-page-status-incorrectly-positionned.zip](https://github.com/squidfunk/mkdocs-material/files/14601574/9.5.13-page-status-incorrectly-positionned.zip)

### Steps to reproduce

1. Create some page in a section,
2. Add a Page status to one of them (at least),
3. Scroll (and shrink the windows if needed) the navigation sidebar,
4. Hover the page status and see the tooltip is incorrectly positionned.

### Browser

Edge

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. Directly related (tooltip positioning):

- #6760
- #6488

**Comment by squidfunk** [comment]
Fixed in e15518cfa588d3dfb8113c1ba90f2c0bc5c02fce – I backported the new tooltip implementation from Insiders in the [refactor/tooltip-positioning](https://github.com/squidfunk/mkdocs-material/tree/refactor/tooltip-positioning) branch! If you could spare a few minutes and check whether this fixes your issue, we can release it in the next days ☺️ Install with:

```
pip install git+https://github.com/squidfunk/mkdocs-material.git@refactor/tooltip-positioning
```

**Comment by Enn83** [comment]
Hi and thanks for fixing this!
I will try this (_on the minimal reproductible example and my repo._) as soon as possible (_tomorrow morning, I don't have access to my doc now_) and I will inform you of the result 😊!

**Comment by Enn83** [comment]
The issue is resolved thanks to your [`refactor/tooltip-positioning`](https://github.com/squidfunk/mkdocs-material/tree/refactor/tooltip-positioning) branch!
I tried in both my documentation and the minimal reproductible example:

![image](https://github.com/squidfunk/mkdocs-material/assets/46783975/af6754ae-c55e-4306-8c92-0128a96a7fd1)

**Comment by squidfunk** [comment]
Thanks! Keeping open until released.

**Comment by squidfunk** [comment]
Released as part of [9.5.18](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.18).
